cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -121,6 +124,12 @@ func CreateStackedEtcdStaticPodManifestFile(client clientset.Interface, manifest
 	}
 
 	fmt.Printf("[etcd] Wrote Static Pod manifest for a local etcd instance to %q\n", kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.Etcd, manifestDir))
```

Comment:
If you don't mind, could you change "local etcd instance" into "local etcd member" (so we are consistent in using always etcd member)